pkgs/development/libraries/proj/default.nix

```diff
@@ -13,7 +14,7 @@
 , python3
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
You don't need `rec` here. All attributes accessable via `finalAttrs`.

Reply:
See above for why I recommended using `rec` instead of `finalAttrs` for the version attribute.

Reply:
OK, I see